packages/react-charting/docs/TestPlans/LineChart/ComponentTests.md

```diff
@@ -0,0 +1,67 @@
+|                             - HideTooltip prop set to “true”                              |                                            Should not mount callout when hideTootip is true                                             |    Enzyme     |
+|                             - HideTooltip prop set to “false”                             |                                              Should mount callout when hideTootip is false                                              |    Enzyme     |
+|                        - onRenderCalloutPerStack prop is not given                        |                                                Should not render onRenderCalloutPerStack                                                |    Pending    |
+|                          - onRenderCalloutPerDataPoint is given                           |                                                Should render onRenderCalloutPerDataPoint                                                |    Pending    |
+|                        - onRenderCalloutPerDataPoint is not given                         |                                              Should not render onRenderCalloutPerDataPoint                                              |    Pending    |
+|                      Test 3: Render calling with respective to props                      |                                                                                                                                         |               |
+|           - No prop changes: Mount line chart and then set the same props again           |                                              Render function should have been called twice                                              |    Enzyme     |
+|               - Prop changes: Mount line chart and then set some other prop               |                                              Render function should have been called twice                                              |    Enzyme     |
+|                                   Test 4: Mouse events                                    |                                                                                                                                         |               |
+|                                  - Mouse over on a line                                   |                                              Should render callout correctly on mouseover                                               | Enzyme & RTL  |
+|                         - Mouse move from one line to other line                          |                                              Should render callout correctly on mouse move                                              | Enzyme & RTL  |
+|                      - Mouse over on a line with customized callout                       |                                              Should render customized callout on mouseover                                              |    Enzyme     |
```

Comment:
what is the difference between - Customized callout on a line and Mouse over on a line with customized callout